index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Philemon portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="page">
+        <header class="menu">
+            <nav> 
+                <img src="images/normal-button.svg" alt="Menu">
+            </nav>
+        </header>
+        <section class="section-container9">
+           <div class="headline">
+            <h1 class="greetings"> Hey there <br>
+                I'm Philemon <br>
+                <span class="job">I'm a Software Developer</span>
+              </h1>
+              <p class="description"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+                <ul class="socials">
+                    <li><a href="#"><img src="images/twitter.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/linkedin.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/medium.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/github.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/angellist.svg" alt="" /></a></li>
+                </ul>
+           </div> 
+           <div class="button">
+            <img src="images/vector-button.svg" alt="button">
+          </div>
+    </div>
+</section>
+    <div class="line-break">
+        <h2 class="works">My Recent <br> Works</h2>
+        </div>
```

Comment:
- [ ] Ouch! this is close but the vertical spacing between the heading of the works section and the project cards seems to be a bit too much when compared with the Figma design so please reduce it. As a front-end developer, you have to pay attention to small details like this as they give your UI a polished look and feel.
- Your design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/3a22ded5-4b1b-4198-90b2-c5abd8df01e3)
- Figma design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/d6169b12-117b-4dfb-b895-3d616844329c)
- [ ] Since the heading "My Recent Works" is part of the works section I'd suggest you place it in the works section element along with the project cards.